scripts/tests/test_bump_version.py

```
@@ -0,0 +1,69 @@
+    dumped = tomlkit.dumps(doc)
+    assert '# pinned for CI' in dumped, "must preserve trailing comment on value node"
+
+
+def test_missing_dependency_no_change() -> None:
+    snippet = '[dependencies]\nbar = "0.1"'
+    doc = tomlkit.parse(snippet)
+    _update_dependency_version(doc, 'foo', '1.2.3')
+    assert tomlkit.dumps(doc).strip() == snippet, "should be a no-op when dependency is absent"
+
+
+def test_update_markdown_versions_updates_toml_fences(tmp_path) -> None:
```

Ruff (0.12.2)

Missing type annotation for function argument `tmp_path`

(ANN001)